ttyplot.c

```diff
@@ -429,29 +520,9 @@ int main(int argc, char *argv[]) {
     signal(SIGWINCH, resize);
     signal(SIGINT, finish);
 
-    char input_buf[4096] = "";
-    size_t input_len = 0;
-
     while(1) {
-        if (sigint_pending) {
-            break;
-        }
-
-        if (sigwinch_pending) {
-            sigwinch_pending = 0;
-
-            endwin();
-            initscr();
-            clear();
-            refresh();
-
-            gethw();
-
-            goto redraw_and_continue;
-        }
-
         // Block until (a) we receive a signal or (b) stdin can be read without blocking
-        // or (c) timeout expires, in oder to reduce use of CPU and power while idle
+        // or (c) timeout expires, in order to reduce use of CPU and power while idle
```

Comment:
(Can we extract a dedicated commit for the typo?)

Reply:
> Can we extract a dedicated commit for the typo fix?

OK. Done. I put the typo-fixing commit first, then the big one. May I add your Co-authored-by to this second commit?